src/js/components/Box/README.md

```diff
@@ -279,6 +279,7 @@ right
 horizontal
 vertical
 all
+between
 {
   color: 
     string
```

Comment:
If it has the possibility of `border.side === between` , I think would be a good Idea to add it here aswell

Reply:
```
....
all
between
{
  color: 
    string
    {
      dark: string,
      light: string
    },
  side: 
    top
    left
    bottom
    right
    horizontal
    vertical
    all
    between
  size: 
...
```

Reply:
https://github.com/grommet/grommet/blob/a3c75c2310ac789120b137f6310a432ffb98fb92/src/js/components/Box/doc.js#L40

Reply:
agreed